src/core/css/SearchResult.scss

```diff
@@ -23,7 +23,15 @@
 }
 
 .SearchResult-main {
+  display: flex;
+  flex-direction: row;
   flex-grow: 1;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.SearchResult-text {
+  width: 90%;
```

Comment:
It does, though for now we've been using `px margin` over `% width` as it keeps the margins consistent across devices and means content can fill a lot of the screen. Plus we have a fairly different design coming up for desktop that will involve a fair number of style overrides/differences anyway, so for now we are keeping things responsive by basically just scaling up the mobile view on desktop, as this v1 is planned to only be the default for mobile devices anyway.

Your point is valid but for this project, for now, please use `px` similar to what was used before to keep things from being full-width. 😄